caffe2/contrib/tensorrt/tensorrt_op_trt.cc

```diff
@@ -54,14 +57,51 @@ TensorRTOp::TensorRTOp(const OperatorDef& operator_def, Workspace* ws)
   {
     auto engine_string =
         OperatorBase::GetSingleArgument<std::string>("serialized_engine", "");
```

Comment:
Can we rename this parameter? `serialized_engine` looks related to Caffe2 engine, albeit in fact uses TRT terminology. SNPEOp uses "model_buffer", which sounds better. This change could be done in a separate PR.

Reply:
It's ok and I can change it here. As the parameter name is internal detail.